wolfcrypt/src/logging.c

```diff
@@ -285,12 +375,70 @@ static void wolfssl_log(const int logLevel, const char* const file_name,
 
 #ifndef WOLFSSL_DEBUG_ERRORS_ONLY
 
+WOLFSSL_API int WOLFSSL_IS_CERT_DEBUG_ON(void)
+{
+    /* see also loggingCertEnabledUnavailable */
+    return loggingCertEnabled;
+}
+/* Unless explicitly disabled with NO_WOLFSSL_DEBUG_CERTS
+ * Certificate debugging available with either regular debugging
+ * DEBUG_WOLFSSL or just certificate debugging: WOLFSSL_DEBUG_CERTS */
+#if (defined(WOLFSSL_DEBUG_CERTS) || defined(DEBUG_WOLFSSL)) && \
+    !defined(NO_WOLFSSL_DEBUG_CERTS)
+    #include <stdarg.h> /* for var args */
+
+    #ifndef WOLFSSL_MSG_CERT_BUF_SZ
+        #define WOLFSSL_MSG_CERT_BUF_SZ 1000
```

Comment:
Why such a large default? This does not conform to our small stack policy. Max 100 bytes on stack.